### PR TITLE
Fix/css modules

### PR DESCRIPTION
**Pull request by gabrielgalc**

### CODE REVIEW

**Comment by GustavoMA** [comment]
You need to run `yarn test -u` in the `react` folder to update snapshots.

File: react/Menu.tsx [hunk 1]
```diff
@@ -42,7 +44,7 @@ const Menu: StorefrontFunctionComponent<MenuSchema> = ({
       <MenuContext.Provider value={menuContext}>
         <nav>
           <ul
-            className={classNames('list flex pl0 mv0', {
+            className={classNames(`${styles.menuContainer} list flex pl0 mv0`, {
```
Comment (Klynger) [inline react/Menu.tsx]: You don't have to use string interpolation here, you could just do

```suggestion
            className={classNames(styles.menuContainer, 'list flex pl0 mv0', {
```

File: react/components/StyledLink.tsx [hunk 2]
```diff
@@ -29,7 +31,7 @@ const StyledLink: FunctionComponent<StyledLinkProps> = props => {
 
   const hasLink = to && to !== '#'
 
-  const linkClassNames = classNames('no-underline pointer', {
+  const linkClassNames = classNames(`${styles.styledLink} no-underline pointer`, {
```
Comment (Klynger) [inline react/components/StyledLink.tsx]: Same

```suggestion
  const linkClassNames = classNames(styles.styledLink, 'no-underline pointer', {
```

File: react/components/StyledLink.tsx [hunk 3]
```diff
@@ -49,7 +51,7 @@ const StyledLink: FunctionComponent<StyledLinkProps> = props => {
 
   return (
     <div
-      className={classNames('mh6', {
+      className={classNames(`${styles.styledLinkContainer} mh6`, {
```
Comment (Klynger) [inline react/components/StyledLink.tsx]:
```suggestion
      className={classNames(styles.styledLinkContainer, 'mh6', {
```

**Pull request merged**